packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -78,9 +104,878 @@ describe("Integration tests", () => {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor1],
+          producerId: tenant1.id,
+        };
+        await addOneEService(eService1, eservices);
+
+        await addOneTenant(tenant2, postgresDB, tenants);
+
+        const descriptor2: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService2: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor2],
+          producerId: tenant2.id,
+        };
+        await addOneEService(eService2, eservices);
+
+        await addOneTenant(tenant3, postgresDB, tenants);
+
+        const descriptor3: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService3: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor3],
+          producerId: tenant3.id,
+        };
+        await addOneEService(eService3, eservices);
+        const tenantsByName = await readModelService.getProducers({
+          producerName: undefined,
+          offset: 0,
+          limit: 3,
+        });
+        expect(tenantsByName.results.length).toBe(3);
+      });
+      it("Should get producers (pagination: offset, limit)", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+
+        const descriptor1: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService1: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor1],
+          producerId: tenant1.id,
+        };
+        await addOneEService(eService1, eservices);
+
+        await addOneTenant(tenant2, postgresDB, tenants);
+
+        const descriptor2: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService2: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor2],
+          producerId: tenant2.id,
+        };
+        await addOneEService(eService2, eservices);
+
+        await addOneTenant(tenant3, postgresDB, tenants);
+
+        const descriptor3: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService3: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor3],
+          producerId: tenant3.id,
+        };
+        await addOneEService(eService3, eservices);
+        const tenantsByName = await readModelService.getProducers({
+          producerName: undefined,
+          offset: 2,
+          limit: 3,
+        });
+        expect(tenantsByName.results.length).toBe(1);
+      });
+    });
```

Comment:
The only tests related to this PR are the highlighted ones, so please focus on this.
The other PRs (already reviewed) have been merged too soon by mistake. 
Sorry about this.
@galales, @beetlecrunch